assets/style.css

```diff
@@ -0,0 +1,781 @@
+  }
+
+  .news {
+    color: #fff !important;
+  }
+
+  .first-nav-ul {
+    padding-left: 10px;
+  }
+
+  /* second-nav-bar */
+
+  .second-nav-links-span span {
+    margin-right: 10px;
+  }
+
+  .second-nav-links-span span a {
+    color: rgb(146, 140, 140);
+  }
+
+  .second-nav-links {
+    width: 97%;
+    font-size: 14px;
+    padding: 18px 12px 0;
+    position: relative;
+  }
+
+  .lens-menu {
+    background-color: red;
+    position: absolute;
+    right: 0;
+  }
+
+  .fa-search {
+    position: absolute;
+    right: 10px;
+  }
+
+  /* SHOWCASE */
+  .showcase-section {
+    grid-template-columns: 70% 30%;
+    gap: 1em;
+  }
+
+  .showcase-logo-div {
+    top: -140px;
+  }
+
+  .showcase-logo {
+    display: inline-block;
+    width: 400px;
+  }
+
+  .release-date {
+    float: right;
+    bottom: 150px;
+    right: 20px;
+  }
+
+  .mobile-showcase-content .release-date {
+    display: block;
+  }
+
+  .main-showcase {
+    height: 400px;
+  }
+
+  .aside-showcase {
+    height: 47%;
+  }
+
+  .main-showcase > div h1 {
+    font-size: 40px;
+  }
+
+  .mobile-showcase-content {
+    bottom: 2px;
+    left: 5px;
+  }
+
+  .aside-showcase > div h1 {
+    font-size: 20px;
+  }
+
+  .aside-showcase > div :nth-last-child(1) {
+    display: none;
+  }
+
+  /* LATEST NEWS SECTION */
+  .latest-news-grid,
+  .latest-funding-contents {
+    grid-template-columns: repeat(4, 1fr);
+    gap: 1em;
+  }
+
+  .latest-news-div {
+    flex-direction: column;
+  }
+
+  span img {
+    width: 37px;
+    height: 9px;
+  }
+
+  .latest-news-div img {
+    width: 100%;
+  }
+
+  /* LATEST FUNDING ROUNDS */
+  .width-768 {
+    display: block;
+  }
+
+  .latest-funding-title-links {
+    display: flex;
+  }
+
+  .funding-links-right {
+    display: flex;
+    flex-direction: row;
+  }
+
+  /* MOST POPULAR */
+  .most-popular-section {
+    grid-template-columns: repeat(3, 1fr);
+    gap: 1em;
+  }
+
+  .hardfork-img {
+    height: 200px;
+    width: 100%;
+    gap: 1em;
+  }
+
+  /* FOOTER */
+  .footer {
+    width: 100%;
+    height: 250px;
+    text-align: center;
+  }
+
+  .footer-links {
+    background-color: #131313;
+    padding: 2.5em 0;
+  }
+
+  .wrapper {
+    margin: 0 auto;
+    max-width: 100%;
+    padding: 0 20px;
+    height: 100%;
+  }
+
+  .footer-links-social,
+  .footer-links-social li a,
+  .footer-links-menu {
+    justify-content: center;
+  }
+
+  .footer-links-social {
+    font-size: 100%;
+    margin-bottom: 15px;
+    vertical-align: baseline;
+  }
+
+  .footer-links-social li a {
+    box-sizing: border-box;
+    background-color: #333;
+    margin: auto 15px;
+    border-radius: 20px;
+    padding: 10px;
+  }
+
+  .footer-links-menu {
+    margin-top: 50px;
+    font-size: 100%;
+    box-sizing: border-box;
+    align-items: center;
+  }
+
+  .footer-links-menu li a {
+    margin: 5px;
+    border: 0;
+    vertical-align: baseline;
+    box-sizing: border-box;
+    align-content: center;
+    flex-wrap: nowrap;
```

Comment:
## Using Inefficient CSS Selectors[1]

When you are using selectors, the longer it is, the higher specificity it will comprise. This can break the CSS cascade and mess up the styles of your site.

This can be avoided by not using IDs within the selectors as they are directly specific.

 - [ ]  Please add a class to `<a>` element` and delete the left unnecessary rules

**Example:**
```CSS
/* before */
.card-1-mobile-text h3 a {

/* after */
.anchortag-class {
```

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_